### DISCUSSION TITLE
Unable to access the domain that a team owned after the team deleted

### DISCUSSION THREAD

**Discussion by simcheolhwan** [vercel/community#1412] [Help]
### Summary

I created a team and the team purchased a domain. And I've deleted the team recently.
Now I want to transfer out the domain but there is no way to access it.
How can I do it?

**Comment by swarnava** [comment]
For security reasons, teams cannot be restored.

However, you can try to recreate a new team with the same name and re-add the domain. Let us know if this results in any errors and we'd be happy to help.

**Reply by simcheolhwan** [reply]
As you said, I tried to create a new team with the same team name, but my domain does not appear. Maybe it's a natural consequence of security? It's weird if the domain I purchased appears when someone else accidentally uses the team name.

**Reply by swarnava** [reply]
What's your team name?
